tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -1090,6 +1090,10 @@ protected:
                (void)expander; // avoid unused variable warnings for older compilers such as gcc 4.9
             }
             trees[slot]->Fill();
+            if (files[slot]->GetFileBytesWritten() > 96L*1024L*1024L) {
```

Comment:
Yes, but that does not trigger the File::Write() that we need to put data on the queue. I need to work on making this work more seamlessly within TBufferMerferFile, but for now that's what we have to do, I guess.